sklearn/metrics/tests/test_pairwise.py

```diff
@@ -1264,11 +1293,11 @@ def test_pairwise_similarity_sparse_output(metric, pairwise_func):
     # should be dense, and equal to K1
     K2 = pairwise_func(X, Y, dense_output=True)
     assert not issparse(K2)
-    assert_array_almost_equal(K1.todense(), K2)
+    assert_allclose(K1.todense(), K2)
```

Comment:
Let's stop using the deprecated `np.matrix` objects in scikit-learn while we are at it.

```suggestion
    assert_allclose(K1.toarray(), K2)
```